R/calculate_aum.R

```diff
@@ -15,5 +15,10 @@ calculate_aum <- function(sector_exposures) {
       asset_portfolio_value = sum(valid_value_usd),
       .groups = "drop"
     )
+
+  if (aum$asset_portfolio_value <= 0) {
+    stop("Calculated implausible asset under management.", call. = FALSE)
```

Comment:
maybe be more precise about the actual error: "Asset under management has negative value. This is not supported by the analysis." something like that. In principle one could imagine a portfolio with lots of short positions that add up to a negative overall value. It should not happen in our analyses, but it is not out of this world to imagine the situation

Reply:
DONE